README.md

```diff
@@ -46,9 +55,10 @@ health-checker [options]
 | `--listener` |  The IP address and port on which inbound HTTP connections will be accepted. | `0.0.0.0:5000`
 | `--log-level` | Set the log level to LEVEL. Must be one of: `panic`, `fatal`, `error,` `warning`, `info`, or `debug` | `info` 
 | `--help` | Show the help screen | | 
+| `--script` | Path to script to run - will PASS if it completes within 5s with a zero exit status | | 
```

Comment:
It's a nit, but any reason PASS is in all caps?

Reply:
Perhaps consider an optional arg `--script-timeout` whose default is set to 5 seconds rather than combining it with `--script`?